build.gradle.kts

```diff
@@ -1,15 +1,21 @@
 buildscript {
     val kotlinVersion by System.getProperties()
     val dokkaVersion = "0.10.1"
+    val androidVersion = "4.0.1"
+    val composeVersion = "0.3.0-build134"
 
     repositories {
         mavenCentral()
         jcenter()
+        google()
+        maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
     }
```

Comment:
`Gradle 6.8` [will have an official way](https://docs.gradle.org/6.8-rc-1/release-notes.html#central-declaration-of-repositories) to declare the repositories for all the projects.

Adding them to `settings.gradle.kts` will let us remove the repositories block in all modules and in the `build.gradle.kts` root file. `buildSrc` have to keep the repositories block with `mavenCentral()`